src/store/undoRedo/index.ts

```diff
@@ -115,15 +124,29 @@ export const actions: ActionTree<IUndoRedoState, IRootState> = {
       flows: rootState.flow,
     })
 
+    let isLastFlowStateFromPersistence = !getters.hasCurrentSnapshot
+    const currentSavedAt = getters.currentSnapshot?.modules.flows?.savedAt
+    const nextSavedAt = nextSnapshot?.modules.flows?.savedAt
+    if (currentSavedAt !== undefined && nextSavedAt !== undefined && currentSavedAt !== nextSavedAt) {
+      isLastFlowStateFromPersistence = true
+    }
```

Comment:
would this be equivalent? 🤔
```
const isLastFlowStateFromPersistence  = nextSavedAt !== currentSavedAt
```
`nextSavedAt` can't be different AND undefined - because we are never clearing it

Reply:
let me reorder the condition if that's more easier to read, no major concern here ;)
But I'd like to keep the `nextSavedAt !== undefined` for precaution

Reply:
re-ordered!